tools/cleed_io/reader.py

```
@@ -0,0 +1,114 @@
+        inp_file (str): Path to .inp file.
+        
+    Returns:
+        Atoms: ASE Atoms object containing Bulk + Overlayer.
+    """
+    
+    # 1. Parse Bulk File
+    a1 = a2 = a3 = None
+    bulk_atoms = []
+    bulk_symbols = []
+    
+    with open(bul_file, 'r') as f:
```

Ruff (0.14.8)

Unnecessary mode argument

Remove mode argument

(UP015)